react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class NewsletterForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { Email: '' };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ Email: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    console.log('email: ', this.state.Email);
+  }
+
+  render() {
+    return (
+          <form onSubmit={this.handleSubmit}>
+              <label>
+                  Email
+                  <input type="text" onChange={this.handleChange} />

```

Comment:
Here the input should have a `value` attribute set to the value from state, in this case `this.state.email`